CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Hi! Thanks for being interested in contributing to Converge. We're really happy
 to have you (we have mini-parties in Slack whenever someone external opens a
-PR!)
+PR!).
```

Comment:
Updated: left the period for the fragment, and removed from the complete sentences.